pkg/kops/kops.go

```diff
@@ -0,0 +1,76 @@
+package kops
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/kubicorn/kubicorn/pkg/logger"
+	"github.com/weaveworks/eksctl/pkg/eks/api"
+	"k8s.io/kops/pkg/resources/aws"
+	"k8s.io/kops/upup/pkg/fi/cloudup/awsup"
+)
+
+// Wrapper for interacting with a kops cluster
+type Wrapper struct {
+	clusterName string
+	cloud       awsup.AWSCloud
+	spec        *api.ClusterConfig
+}
+
+// NewWrapper constructs a kops wrapper for a given EKS cluster config and kops cluster
+func NewWrapper(cfg *api.ClusterConfig, kopsClusterName string) (*Wrapper, error) {
+	cloud, err := awsup.NewAWSCloud(cfg.Region, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	return &Wrapper{kopsClusterName, cloud, cfg}, nil
+}
+
+func (k *Wrapper) isOwned(t *ec2.Tag) bool {
+	return *t.Key == "kubernetes.io/cluster/"+k.clusterName && *t.Value == "owned"
+}
+
+// UseVPC finds VPC and subnets that give kops cluster uses and add those to EKS cluster config
+func (k *Wrapper) UseVPC() error {
```

Comment:
```
UseVPC(spec api.ClusterConfig)
```

Reply:
I guess a nice alternative would be `GetVPC() *api.ClusterVPC`, which would lead to some refactoring, but that maybe a wise idea anyway, we'd probably want to group some related fields inside of `api.ClusterConfig` anyway (most notably nodegroup stuff).